src/orquestra/sdk/_ray/_build_workflow.py

```diff
@@ -213,14 +212,14 @@ def _make_ray_dag_node(
             see ArgumentUnwrapper.
         kwargs_artifact_nodes: a map of keyword arg name to artifact node
             see ArgumentUnwrapper.
-        project_dir: the working directory the workflow was submitted from.
         user_fn_ref: function reference for a function to be executed by Ray.
             if None - executes data aggregation step.
         output_metadata: output metadata for the user task function.
             Keeps number of outputs and if the output is subscriptable.
         dry_run: Run the task without actually executing any user code.
             Useful for testing infrastructure, dependency imports, etc.
     """
+    current_path = Path.cwd()
```

Comment:
**thought**: it looks we're safe for now, but we'll have to be careful about where this code is being executed. If it was wrapped in some Ray remote, `Path.cwd()` is likely to be resolved to PWD at the time of running `orq up` which might be different from where the project files need to be loaded

Reply:
**thought 2**: maybe some time in the future we can get of the `sys.path` manipulation in the first place?

Reply:
That would be great, but Im honestly not sure we are even able to do that due to how we rely on users code to be on the path to launch scripts and other stuff